src/state/notifications/sagas.ts

```diff
@@ -68,7 +79,63 @@ export function* setNotificationEmailSaga(action: SetNotificationEmailAction | u
   }
 }
 
+export function* subscribeWalletSaga(action: SubscribeWalletAction) {
+  const { payload } = action as SubscribeWalletAction;
+  try {
+    const response: { session_token: string; result: 'success' | 'error' } = yield call(subscribeEmail, payload);
+    if (response.session_token) {
+      yield put(subscribeWalletSuccess(response.session_token));
+    }
+  } catch (error) {
+    // TODO
```

Comment:
imho should be done in this PR

Reply:
depends if you want to have it in develop now

Reply:
adding one action to redux shouldn't take much time